db/seeds.rb

```diff
@@ -1,7 +1,528 @@
+  },
+  {
+    rating: 2,
+    product_id: 15,
+    comment: "Vision isn't as good as expected."
+  },
+  {
+    rating: 3,
+    product_id: 15,
+    comment: "I love using this product but the battery doesn't last very long."
+  },
+  {
+    rating: 3,
+    product_id: 15,
+    comment: "I love using this product."
+  },
+  {
+    rating: 3,
+    product_id: 16,
+    comment: "I love using this product."
+  },
+  {
+    rating: 3,
+    product_id: 16,
+    comment: "It's okay."
+  },
+  {
+    rating: 3,
+    product_id: 16,
+    comment: "I love using this product."
+  },
+  {
+    rating: 4,
+    product_id: 17,
+    comment: "I love using this product."
+  },
+  {
+    rating: 3,
+    product_id: 17,
+    comment: "I like apple watch better."
+  },
+  {
+    rating: 4,
+    product_id: 17,
+    comment: "I love using this product."
+  },
+  {
+    rating: 2,
+    product_id: 18,
+    comment: "Not a fan."
+  },
+  {
+    rating: 4,
+    product_id: 18,
+    comment: "No more ruined nails!"
+  },
+  {
+    rating: 3,
+    product_id: 18,
+    comment: "It's okay but totally unnecessary."
+  },
+  {
+    rating: 3,
+    product_id: 19,
+    comment: "It's okay but totally unnecessary."
+  },
+  {
+    rating: 3,
+    product_id: 19,
+    comment: "I like apple watch better."
+  },
+  {
+    rating: 2,
+    product_id: 19,
+    comment: "Not a fan."
+  },
+  {
+    rating: 4,
+    product_id: 20,
+    comment: "I really love using this product."
+  },
+  {
+    rating: 5,
+    product_id: 20,
+    comment: "So helpful for dusting."
+  },
+  {
+    rating: 4,
+    product_id: 20,
+    comment: "I like it."
+  },
+  {
+    rating: 3,
+    product_id: 21,
+    comment: "I like it."
+  },
+  {
+    rating: 4,
+    product_id: 21,
+    comment: "This product is cool."
+  },
+  {
+    rating: 4,
+    product_id: 22,
+    comment: "This product is cool."
+  },
+  {
+    rating: 5,
+    product_id: 22,
+    comment: "OMG MY NEW FAVORITE THING I USE IT EVERYDAY"
+  },
+  {
+    rating: 4,
+    product_id: 23,
+    comment: "This product is cool."
+  },
+  {
+    rating: 5,
+    product_id: 23,
+    comment: "OMG MY NEW FAVORITE THING I USE IT EVERYDAY"
+  },
+  {
+    rating: 4,
+    product_id: 24,
+    comment: "This product is cool."
+  },
+  {
+    rating: 2,
+    product_id: 24,
+    comment: "You get what you pay for."
+  },
+  {
+    rating: 2,
+    product_id: 25,
+    comment: "You get what you pay for."
+  },
+  {
+    rating: 4,
+    product_id: 25,
+    comment: "This product is cool."
+  },
+]
+
+categories = [
+  { name: "art" },
+  { name: "food" },
+  { name: "gadgets" },
+  { name: "pets" },
+  { name: "home" },
+  { name: "clothing" },
+  { name: "software" }
+]
+
+products.each do |product|
+  Product.create(product)
+end
+
+categories.each do |category|
+  Category.create(category)
+end
+
+reviews.each do |review|
+  Review.create(review)
+end
+
+Product.find(1).categories << Category.find(4)
```

Comment:
Even though this is a seed file, it would still be good to ensure you are only querying the database as much as you need. It might make sense to store each of the categories in a variable and shovel that data into the products categories. By using the `Category.find` over and over I think it will take longer than needed. That being said, it is probably not noticeable in this database size and this size of seeds, but I think it's still good to keep in mind.